sources/packages/web/src/services/http/StudentApi.ts

```diff
@@ -153,7 +154,9 @@ export class StudentApi extends HttpBaseClient {
    */
   async getStudentFileDetails(
     studentId: number,
```

Comment:
Not part of this ticket but could you just add `studentId` to the comments please?